### ISSUE TITLE
Implement a `default` command that extends an instance with default values from the schema

### ISSUE THREAD

**Issue by jviotti** [sourcemeta/jsonschema#200]

**Comment by RoaaFouda** [comment]
Hello @jviotti, can i work on this? 
Also, just to make sure I understood the issue correctly, this is a command that fills the missing properties of the instance with the default values from the schema right?

**Comment by jviotti** [comment]
Yeah, go ahead. No need to ask! The idea is to use Blaze (https://github.com/sourcemeta/blaze), which is already a dependency of this project, evaluate a schema against an instance in `Exhaustive` mode (https://blaze.sourcemeta.com/group__evaluator.html#ggad8ce1334da5d759575d390a3cd6eabf0afaf34f09ea1e93bfbf591e19dc0dfb9f), which will collect annotations, collect annotations from `default` (using the evaluation callback: https://blaze.sourcemeta.com/group__evaluator.html#ga7fd1ab25e35cc6e216e604fad0b24b2e), and if needed, add them back to the instance.

We don't want to mutate an instance is place though. Instead, create a copy of the instance in memory, add the information, and print it to `stdout`.

**Comment by RoaaFouda** [comment]
Okay I'll start working on it and let you know if I have any questions, Thanks a lot for the clarification!

**Comment by jviotti** [comment]
I attempted an implementation, and turns out this is not very doable right now (at least not with annotations), given https://github.com/json-schema-org/json-schema-spec/issues/867. I'll close it for now and we can revisit in the future.

**Comment by jviotti** [comment]
For the record, here is another issue describing this transformation and other interesting ones we should keep in mind for the future: https://github.com/json-schema-org/community/issues/318
